File: URI-cpp/1065.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
    int val = 0,n;

    for (int i = 0; i < 5; i++)
    {
        cin >> n;

        if ((n%2)==0)
        {
            val++;
        }
    }
    
    cout << val << " valores pares" << endl;

    return 0;
}

File: URI-py/1065.py
# -*- coding: utf-8 -*-
pares = 0
lista = []
lista.append(int(input()))
lista.append(int(input()))
lista.append(int(input()))
lista.append(int(input()))
lista.append(int(input()))
for i in lista:
    if (i%2)==0:
        pares += 1
print("%d valores pares"%pares)
